### ISSUE TITLE
Create a sample app

### ISSUE THREAD

**Issue by wmzhai** [nicolaslopezj/meteor-react-form#3]
I think this project is a great job for meteor react developers.
But I found it's hard to follow the README document to use this package.
Could you please provide a little sample app which includes basic insert and update.

Thanks!

**Comment by nicolaslopezj** [comment]
The sample app will be ASAP.
I will change the title of this issue, to track it

**Comment by wmzhai** [comment]
very nice, you are doing a great job :)

**Comment by thearabbit** [comment]
Great :+1:

**Comment by nicolaslopezj** [comment]
I created a npm version of this package in other repo

https://github.com/nicolaslopezj/simple-react-form
